.clang-tidy

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
How did you generate this file? Were these all intentionally set or the result of some default setting?